extension/src/test/suite/vscode/recommend.test.ts

```diff
@@ -1,13 +1,22 @@
 import { join } from 'path'
-import { afterEach, beforeEach, describe, it, suite } from 'mocha'
+import { afterEach, before, beforeEach, describe, it, suite } from 'mocha'
 import { expect } from 'chai'
 import { MessageItem, Uri, window } from 'vscode'
 import { restore, stub } from 'sinon'
 import { closeAllEditors } from '../util'
 import { dvcDemoPath } from '../../util'
 import * as Extensions from '../../../vscode/extensions'
+import { recommendRedHatExtensionOnce } from '../../../vscode/recommend'
 
 suite('Recommend Test Suite', () => {
+  const openFileInEditor = (fileName: string) =>
+    window.showTextDocument(Uri.file(join(dvcDemoPath, fileName)))
+
+  before(async () => {
+    await openFileInEditor('dvc.lock') // clear any existing recommendation
```

Comment:
[F] Because we opened a `dvc.yaml` in another integration test this started failing. That is just bad testing practices. This change keeps things independent.